src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/StorageAnalyticsLogEntry.cs

```diff
@@ -139,23 +137,19 @@ public BlobPath ToBlobPath()
             else
             {
                 // assuming key is "/account/container/blob"
-                int startPos = RequestedObjectKey.Length > 1 ? RequestedObjectKey.IndexOf('/', 1) : -1;                
+                int startPos = RequestedObjectKey.Length > 1 ? RequestedObjectKey.IndexOf('/', 1) : -1;
                 path = startPos != -1 ? RequestedObjectKey.Substring(startPos + 1) : String.Empty;
             }
 
             if (String.IsNullOrEmpty(path))
             {
-                throw new FormatException("Failed to parse RequestedObjectKey property of the log entry. " +
-                    "It should be in one of the supported formats: " +
-                    @"""https://account.blob.core.windows.net/container/blob"", or" +
-                    @"""/account/container/blob""");
+                return null;
```

Comment:
I don't think we can simply return null here - there are upstream callers of this method that assume the returned value will not be null (e.g. BlobLogListener).